internal/cli/alerts/alerts.go

```diff
@@ -35,6 +35,8 @@ func Builder() *cobra.Command {
 		ListBuilder(),
 		AcknowledgeBuilder(),
 		UnacknowledgeBuilder(),
+		EnableBuilder(),
+		DisableBuilder(),
```

Comment:
these should be added to https://github.com/mongodb/mongodb-atlas-cli/blob/c45e940f6ecc34b687b1df479c2bb8115514f471/internal/cli/alerts/settings/settings.go#L164-L170 if I'm getting the request correctly 
```suggestion
```